src/protagonist/API/Features/Customer/Requests/AssetUpdater.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections.Generic;
+using API.Features.Customer.Validation;
+using DLCS.Core.Collections;
+using DLCS.Model;
+using DLCS.Model.Assets;
+using DLCS.Repository;
+using DLCS.Repository.Assets;
+using Hydra.Collections;
+using Hydra.Model;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json.Linq;
+
+namespace API.Features.Customer.Requests;
+
+public interface IAssetUpdater
+{
+    /// <summary>
+    /// Updates assets with new values from all image requests
```

Comment:
Small thing, for name and comment: this is a set kind of update. Bulk path, or field patch rather than just `AssetUpdater`. 

re: comment - in future this might not come from "all image request", might be other endpoints that allow the bulk patching of some other asset (e.g. adding tags to a space or something).